salt/journal/config/etc-nginx-sites-available-journal.conf

```diff
@@ -211,8 +211,31 @@ server {
 
     {% endif %}
 
+    {% if pillar.journal.about_url %}
```

Comment:
build is failing because this doesn't appear the pillar data:
https://github.com/elifesciences/journal-formula/blob/master/salt/pillar/journal.sls

formulas are self contained and we can overlay additional pillar data using builder-private and builder-configuration. It follows a deep-merge algorithm. builder-private data overrides builder-configuration data which overrides the formula's data.

Reply:
added `about_url` and merged your builder-config change too. build is green